infrastructure/cluster/flux-v2/jhub/jhub-release.yaml

```diff
@@ -271,7 +175,7 @@ spec:
         RUCIO_DESTINATION_RSE: "CERN-EOS"
         RUCIO_RSE_MOUNT_PATH: "/eos/escape"
         RUCIO_PATH_BEGINS_AT: "5"
-        RUCIO_CA_CERT: "/opt/cert.pem" # this could be set in the image as well
+        RUCIO_CA_CERT: "/certs/rucio_ca.pem"
```

Comment:
Double checking with both @goseind @egazzarr 
The env var `RUCIO_CA_CERT` and the `ca_cert = /opt/cert.pem` option inside the `rucio.cfg` file should be the same, right ?